src/routes/gallery/find.js

```diff
@@ -0,0 +1,44 @@
+import Bluebird from 'bluebird'
+
+import models from '../../models'
+
+export default {
+
+  method: 'get',
+  path: '/galler(y|ies)',
+
+  handler(request, response) {
+
+    return models.Gallery.find()
+      .then((galleries) => {
+
+        return Bluebird.map(galleries, (gallery) => {
+
+          // @NOTE: for phase 1, we're just going to return all public records
```

Comment:
Is this comment still relevant since you've added those knobs on the front-end?

Reply:
Yes. Right now this logic is just grabbing all explicitly marked Records for any gallery created by the user, when in reality there wouldn't be an extra step to retrieve Records here, as they will have already populated the codexRecords array via the front-end "create a new gallery" UI that doesn't exist yet. This is kind of a stopgap measure for the single-gallery V1 stuff to work.